packages/react/src/components/List/List.test.jsx

```diff
@@ -1013,5 +1041,37 @@ describe('List', () => {
       );
       expect(screen.getByRole('checkbox')).toBeDisabled();
     });
+
+    it('should not trigger search onChange when hasFastSearch:false', () => {
```

Comment:
are these duplicate test as line 481?

Reply:
These are with `isVirtualList:true`. They're in their own `describe` block.